app/models/mediaflux/http/request.rb

```diff
@@ -37,21 +49,14 @@ def self.mediaflux_port
         Rails.configuration.mediaflux["api_port"].to_i
       end
 
-      # Constructs a new HTTP client for usage with the Mediaflux API
-      # @return [Net::HTTP]
-      def self.build_http_client
-        Net::HTTP.new(mediaflux_host, mediaflux_port)
```

Comment:
As this uses Class variables (rather than arguments), should this be memoized?

```ruby
@http_client =|| Net::HTTP.new(mediaflux_host, mediaflux_port)
```